packages/core/src/language-server/glint-language-server.ts

```diff
@@ -48,7 +48,7 @@ interface TransformedOffsets {
 }
 
 export default class GlintLanguageServer {
-  private service: ts.LanguageService;
+  readonly service: ts.LanguageService;
```

Comment:
We make this readonly instead of private so that we can pull it off the language server and return it as part of the `ProjectAnalysis`